spec/sections/standard.adoc

```diff
@@ -127,14 +128,14 @@ time. Reference <<environment-variables>> for clarification on what the implemen
 objects and MAY contain any number of elements. There is no other standard restriction on the array size.
 
 .Scalar Elements
-The Scalar objects MAY include any arbitrary custom resource name, but there are reserved resources `cpu`, `disk` and
-`mem` that have special meaning and SHOULD be populated by all Seed compliant images.
+The Scalar objects MAY include any arbitrary custom resource name, but there are reserved resources `cpu`, `disk`,
+`mem` and `sharedMem` that have special meaning and SHOULD be populated by all Seed compliant images.
```

Comment:
sharedMem will only be applicable to some algorithms, so I don't think providing it is a "SHOULD" for all Seed images.